crates/network/src/transaction/builder.rs

```diff
@@ -38,6 +39,44 @@ impl TransactionBuilderError {
     }
 }
 
+/// Wrapper for [`InvalidTransactionRequestError`]s.
+#[derive(Debug)]
+pub struct InvalidTransactionRequestErrors(pub Vec<InvalidTransactionRequestError>);
```

Comment:
it seems like this newtype is not adding any functionality, or preventing any common mistakes. It can be omitted, with the buildererror variant containing an unwrapped Vec

Reply:
It's for the `Display` impl

Reply:
let's manually impl that display?